docs/tasks.rst

```diff
@@ -2139,6 +2139,20 @@ Command Syntax
 
 
 
+**get_existing_recordtypes**
```

Comment:
You'll have a merge conflict here because we're actually removing this file from the source tree to smooth documentation generation.